interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -1969,21 +1969,16 @@ public function logCCDA($pid, $encounter, $content, $time, $status, $user_id, $v
     {
         $content    = base64_decode($content);
         $file_path  = '';
-        $docid = '';
-        $revid = '';
+        $couch_id   = array();
         if ($GLOBALS['document_storage_method'] == 1) {
-            $couch = new CouchDB();
-            $docid = $couch->createDocId('ccda');
-            if ($GLOBALS['couchdb_encryption']) {
-                $encrypted = 1;
-                $cryptoGen = new CryptoGen();
-                $resp = $couch->save_doc(['_id' => $docid, 'data' => $cryptoGen->encryptStandard($content, null, 'database')]);
-            } else {
-                $encrypted = 0;
-                $resp = $couch->save_doc(['_id' => $docid, 'data' => base64_encode($content)]);
-            }
-            $docid = $resp->id;
-            $revid = $resp->rev;
+            $data = array(
+                'data'      => base64_encode($content),
+                'pid'       => $pid,
+                'encounter' => $encounter,
+                'mimetype'  => 'text/xml'
+            );
+            $couch    = \Documents\Plugin\Documents::couchDB();
+            $couch_id = \Documents\Plugin\Documents::saveCouchDocument($couch, $data);
```

Comment:
looks like recent code is getting reverted

Reply:
looks like just in this script and in the below EncountermanagerTable.php script

Reply:
i can sort this out quickly after in codebase (if wanna keep the momentum going :) )

Reply:
thanks Eagle Eye but, i'll go ahead and fix so I don't have to pull in again as I have many current changes.
Who comes up with some of this PSR nonsense!! PHP has no problem with compounding name spaces as many reasonable levels as I wish.... I hate arbitrary junk like this. Or maybe I just hate being caught at it!:)